File: server/languages/javascript/source/template/TestCaseTemplate.js
try {
	let _______lastResult = $$__INVOCATION__$$;
	_______unitTestResultVectors[_______unitTestResultVectors.length-1]["result"].push({"output": `${_______lastResult}`, "type": _______lastResult.constructor.name, "exception": false});
}
catch (err) {
	_______unitTestResultVectors[_______unitTestResultVectors.length-1]["result"].push({"output": `${err}`, "type": err.constructor.name, "exception": true});
}

File: server/languages/python/module/data/TestCaseTemplate.py
try:
	_______lastResult = $$__INVOCATION__$$
	_______unitTestResultVectors[-1]["result"].append({"output":repr(_______lastResult), "type":type(_______lastResult).__name__, "exception":False})
except Exception as err:
	_______unitTestResultVectors[-1]["result"].append({"output":repr(err), "type":type(err).__name__, "exception":True})
